channeldb/graph.go

```diff
@@ -118,6 +118,18 @@ var (
 	// edge's participants.
 	zombieBucket = []byte("zombie-index")
 
+	// disabledEdgePolicyBucket is a sub-bucket of the main edgeBucket bucket
```

Comment:
No migration to populate the bucket for all the _existing_ disabled channels? Once nodes update, even if all the channels in the graph are currently disabled, the `DisabledChannelIDs` method will return an empty slice. 

This also means that any double disable channels which exist (lets say the node is dormant right now but has that double disable state, and hasn't yet ticked to prune again), will only be pruned once they get old enough, rather than immediately.

Reply:
Nice catch! I wonder if writing this migration, even if it's a simple one, will actually be worth it though. It seems like this would only really affect nodes that haven't started up in a while. Path finding already ignores disabled edges, but the larger graph will likely incur a performance hit (though we don't have any concrete results).

Reply:
This seems okay, new nodes will see the effects immediately while existing nodes will see them fully after ~2 weeks.